.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "pelutils/hashmap.c"]
```

Comment:
Du tilføjer den som en dependency og så laver du det om i et andet commit til at være under `ds`, men du behøver ikke fixe det, hvis det er frygteligt.

Imponerende, at dette virker!

Reply:
Jeg er også overrasket over, hvor gnidningsfrit, det er.

Jeg har fucket godt og grundigt rundt i historikken - aka jeg bør virkeligt stoppe med at lave så mange commits ad gangen, for det gør det alt for nederen at rette de tidligere commits.
Jeg squasher bare alle commits, for jeg gider ikke lægge alle rettelserne ind de rigtige steder heller.